pennylane/qchem/vibrational/__init__.py

```diff
@@ -2,6 +2,15 @@
 This submodule provides the functionality to calculate vibrational Hamiltonians.
 """
 
+from .taylor_ham import (
```

Comment:
From what I saw in the DSO repo, it seems like they originally had a flag to return these parts. I think it is okay to leave them as public functions, since perhaps there is a use-case to be able to simply get only the an/harmonic parts for some reason (e.g. for fitting one-mode terms).

Reply:
The fitting functions (e.g. _fit_onebody) are already private.

Reply:
So a user won't need this for fitting in that case as fitting is private as well.